client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -0,0 +1,48 @@
+{{#let @form as |Form|}}
+  <Ui::Question class="fieldset relative scale-fade-in" as |Q|>
+    <Q.legend>
+      <strong>
+        <span class="label">
```

Comment:
The `<strong>` tag isn't necessary here, since the `<Q.Legend>` component includes a class to make the text bold. You only need to specify font weight in this component if you want to override the bold — e.g. `<span class="text-weight-normal">`.